### ISSUE TITLE
bug: unable to run checkly cli with esm project

### ISSUE THREAD

**Issue by devunt** [checkly/checkly-cli#619]
### Node.js version

v19.8.1

### NPM version

pnpm v8.0.0

### @checkly/cli version

0.4.9

### Steps to reproduce

Using `@checkly/cli` in ESM typescript project produces the following error:
```
~/w/rezero develop• ❱ pnpm checkly test
    Error: Error loading file /Users/devunt/workspace/rezero/checkly.config.ts
    Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /Users/devunt/workspace/rezero/checkly.config.ts
    require() of ES modules is not supported.
    require() of /Users/devunt/workspace/rezero/checkly.config.ts from
    /Users/devunt/workspace/rezero/node_modules/.pnpm/@checkly+cli@0.4.9_typescript@5.0.2/node_modules/@checkly/cli/dist/services/util.js is an ES module file as it is
     a .ts file whose nearest parent package.json contains "type": "module" which defines all .ts files in that package scope as ES modules.
    Instead change the requiring code to use import(), or remove "type": "module" from /Users/devunt/workspace/rezero/package.json.

        at createErrRequireEsm
    (/Users/devunt/Library/pnpm/global/5/.pnpm/ts-node@10.9.1_jh3j7gf3qptnaxtpzk3sx5vjku/node_modules/ts-node/dist-raw/node-internal-errors.js:46:15)
        at assertScriptCanLoadAsCJSImpl
    (/Users/devunt/Library/pnpm/global/5/.pnpm/ts-node@10.9.1_jh3j7gf3qptnaxtpzk3sx5vjku/node_modules/ts-node/dist-raw/node-internal-modules-cjs-loader.js:584:11)
        at Object.require.extensions.<computed> [as .ts]
    (/Users/devunt/Library/pnpm/global/5/.pnpm/ts-node@10.9.1_jh3j7gf3qptnaxtpzk3sx5vjku/node_modules/ts-node/src/index.ts:1610:5)
        at Module.load (node:internal/modules/cjs/loader:1133:32)
        at Function.Module._load (node:internal/modules/cjs/loader:972:12)
        at Module.require (node:internal/modules/cjs/loader:1157:19)
        at require (node:internal/modules/helpers:119:18)
        at loadTsFile (/Users/devunt/workspace/rezero/node_modules/.pnpm/@checkly+cli@0.4.9_typescript@5.0.2/node_modules/@checkly/cli/src/services/util.ts:41:33)
        at processTicksAndRejections (node:internal/process/task_queues:95:5)
        at async loadChecklyConfig
    (/Users/devunt/workspace/rezero/node_modules/.pnpm/@checkly+cli@0.4.9_typescript@5.0.2/node_modules/@checkly/cli/src/services/checkly-config-loader.ts:87:14)
```

In my `package.json`, there are `"type": "module"` and I have the corresponding configurations in `tsconfig.json` which is following:
```
		"moduleResolution": "node",
		"module": "esnext",
		"target": "esnext",
```

Does `@checkly/cli` only works in commonjs projects?

### What is expected?

`@checkly/cli` runs.

### What is actually happening?

Instead `@checkly/cli` produces error.

**Comment by muratso** [comment]
I believe this, in essence, is a duplicate of https://github.com/checkly/checkly-cli/issues/616

**Comment by tnolet** [comment]
@devunt @muratso thanks for reporting. We are looking into this. @devunt would you mind pasting your full `package.json` and `tsconfig.json`?

**Comment by devunt** [comment]
@tnolet It's loooong.

```jsonc
// package.json
{
  "name": "crepe",
  "version": "0.0.1",
  "private": true,
  "type": "module",
  "scripts": {
    "install-hooks": "husky install",
    "dev": "vite dev",
    "build": "pnpm run bootstrap && vite build",
    "codegen": "houdini generate --pull-schema",
    "bootstrap": "svelte-kit sync && houdini generate",
    "lint": "eslint --max-warnings=0 .",
    "format": "eslint --fix .",
    "check": "svelte-check --fail-on-warnings",
    "test": "playwright test"
  },
  "devDependencies": {
    "@envelop/core": "^3.0.6",
    "@graphql-yoga/plugin-graphql-sse": "^1.4.0",
    "@iconify-json/ri": "^1.1.5",
    "@iconify/utils": "^2.1.5",
    "@planetscale/database": "^1.6.0",
    "@playwright/test": "^1.32.1",
    "@pothos/core": "^3.29.0",
    "@pothos/plugin-simple-objects": "^3.6.7",
    "@pothos/plugin-with-input": "^3.10.0",
    "@sveltejs/adapter-vercel": "^2.4.1",
    "@sveltejs/kit": "^1.15.0",
    "@typescript-eslint/eslint-plugin": "^5.57.0",
    "@typescript-eslint/parser": "^5.57.0",
    "@unocss/eslint-plugin": "^0.50.6",
    "@unocss/preset-mini": "^0.50.6",
    "@unocss/reset": "^0.50.6",
    "@vercel/analytics": "^0.1.11",
    "eslint": "^8.37.0",
    "eslint-config-prettier": "^8.8.0",
    "eslint-import-resolver-typescript": "^3.5.4",
    "eslint-plugin-import": "^2.27.5",
    "eslint-plugin-prettier": "^4.2.1",
    "eslint-plugin-simple-import-sort": "^10.0.0",
    "eslint-plugin-svelte": "^2.24.0",
    "graphql": "^16.6.0",
    "graphql-scalars": "^1.21.3",
    "graphql-sse": "^2.1.1",
    "graphql-yoga": "^3.8.0",
    "houdini": "^1.1.5",
    "houdini-svelte": "^1.1.5",
    "husky": "^8.0.3",
    "lint-staged": "^13.2.0",
    "prettier": "^2.8.7",
    "prettier-plugin-svelte": "^2.10.0",
    "sass": "^1.60.0",
    "svelte": "^3.58.0",
    "svelte-check": "^3.1.4",
    "svelte-preprocess": "^5.0.3",
    "typescript": "^5.0.3",
    "unocss": "^0.50.6",
    "vite": "^4.2.1",
    "web-vitals": "^3.3.0",
    "ws": "^8.13.0"
  }
}
```

```jsonc
// tsconfig.json
{
  "extends": "./.svelte-kit/tsconfig.json",
  "compilerOptions": {
    "forceConsistentCasingInFileNames": true,
    "rootDirs": [".", "./.svelte-kit/types", "./$houdini/types"],
    "skipLibCheck": true,
    "strict": true
  }
}
```

```jsonc
// .svelte-kit/tsconfig.json
{
	"compilerOptions": {
		"paths": {
			"$houdini": [
				"../$houdini"
			],
			"$houdini/*": [
				"../$houdini/*"
			],
			"$lib": [
				"../src/lib"
			],
			"$lib/*": [
				"../src/lib/*"
			]
		},
		"rootDirs": [
			"..",
			"./types"
		],
		"importsNotUsedAsValues": "error",
		"isolatedModules": true,
		"preserveValueImports": true,
		"lib": [
			"esnext",
			"DOM",
			"DOM.Iterable"
		],
		"moduleResolution": "node",
		"module": "esnext",
		"target": "esnext",
		"ignoreDeprecations": "5.0"
	},
	"include": [
		"ambient.d.ts",
		"./types/**/$types.d.ts",
		"../vite.config.ts",
		"../src/**/*.js",
		"../src/**/*.ts",
		"../src/**/*.svelte",
		"../tests/**/*.js",
		"../tests/**/*.ts",
		"../tests/**/*.svelte"
	],
	"exclude": [
		"../node_modules/**",
		"./[!ambient.d.ts]**",
		"../src/service-worker.js",
		"../src/service-worker.ts",
		"../src/service-worker.d.ts"
	]
}
```

**Comment by umutuzgur** [comment]
@devunt this should be fixed with the 4.0.7 release. We now do support mjs imports

**Comment by alvarlagerlof** [comment]
> @devunt this should be fixed with the 4.0.7 release. We now do support mjs imports

Does this work for TypeScript as well?

**Comment by evdama** [comment]
Hi folks, I'm running into this issue as well

```
sa@mst: ~/0/edm  |main U:2 ?:4 ✗| npx checkly login
✔ Do you want to log in or sign up to Checkly? › I want to log in with an existing Checkly account
✔ Do you want to open a browser window to continue with login? … yes
Successfully logged in as Mark
Welcome to the Checkly CLI
sa@mst: ~/0/edm  |main U:2 ?:4 ✗| checkly -v
checkly/4.0.8 darwin-x64 node-v20.3.0
sa@mst: ~/0/edm  |main U:2 ?:4 ✗| npx checkly test
Parsing your project... !
    Error: Error loading file /Users/sa/0/edm/checkly.config.ts
    Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /Users/sa/0/edm/checkly.config.ts
    require() of ES modules is not supported.
    require() of /Users/sa/0/edm/checkly.config.ts from /Users/sa/0/edm/node_modules/checkly/dist/services/util.js is an ES module file as it is a .ts file 
    whose nearest parent package.json contains "type": "module" which defines all .ts files in that package scope as ES modules.
    Instead change the requiring code to use import(), or remove "type": "module" from /Users/sa/0/edm/package.json.

        at createErrRequireEsm (/Users/sa/0/edm/node_modules/ts-node/dist-raw/node-internal-errors.js:46:15)
        at assertScriptCanLoadAsCJSImpl (/Users/sa/0/edm/node_modules/ts-node/dist-raw/node-internal-modules-cjs-loader.js:584:11)
        at Object.require.extensions.<computed> [as .ts] (/Users/sa/0/edm/node_modules/ts-node/src/index.ts:1610:5)
        at Module.load (node:internal/modules/cjs/loader:1113:32)
        at Function.Module._load (node:internal/modules/cjs/loader:960:12)
        at Module.require (node:internal/modules/cjs/loader:1137:19)
        at require (node:internal/modules/helpers:121:18)
        at /Users/sa/0/edm/node_modules/checkly/src/services/util.ts:65:54
        at processTicksAndRejections (node:internal/process/task_queues:95:5)
        at async loadTsFile (/Users/sa/0/edm/node_modules/checkly/src/services/util.ts:65:33)    
sa@mst: ~/0/edm  |main U:2 ?:4 ✗|
```
It's a sveltekit project running on vercel. Here's the package.json

...
{
  "name": "edm",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "dev": "vite dev",
    "build": "vite build",
    "preview": "vite preview",
    "test": "playwright test",
    "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
    "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
    "test:unit": "vitest",
    "lint": "prettier --plugin-search-dir . --check . && eslint .",
    "format": "prettier --plugin-search-dir . --write ."
  },
  "devDependencies": {
    "@playwright/test": "^1.35.1",
    "@skeletonlabs/skeleton": "^1.7.1",
    "@sveltejs/adapter-vercel": "^3.0.1",
    "@sveltejs/kit": "^1.20.4",
    "@tailwindcss/forms": "^0.5.3",
    "@tailwindcss/typography": "^0.5.9",
    "@typescript-eslint/eslint-plugin": "^5.60.0",
    "@typescript-eslint/parser": "^5.60.0",
    "autoprefixer": "^10.4.14",
    "eslint": "^8.43.0",
    "eslint-config-prettier": "^8.8.0",
    "eslint-plugin-svelte": "^2.31.0",
    "postcss": "^8.4.24",
    "prettier": "^2.8.8",
    "prettier-plugin-svelte": "^2.10.1",
    "svelte": "^3.59.1",
    "svelte-check": "^3.4.3",
    "tailwindcss": "^3.3.2",
    "tslib": "^2.5.3",
    "typescript": "latest",
    "vite": "^4.3.9",
    "vitest": "^0.32.2",
    "checkly": "latest",
    "ts-node": "latest"
  },
  "type": "module",
  "dependencies": {
    "@vercel/analytics": "^1.0.1"
  }
}
...
